team_admin/kit_shipping.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Kit Shipping
+---
+
+# Kit Shipping
+
+This page provides details of how to ship a Student Robotics kit. Please only ship a Student Robotics kit if instructed to do so by the [Kit Logistics Coordinator](mailto:logistics@studentrobotics.org). If you have a kit that you need to ship to us please follow the instructions in the [Packaging](#packaging) section. Once you have your kit packaged, or are looking to get a kit collected from us to be delivered to you, please follow the instructions in the [Arranging a Courier](#arranging-a-courier) section.
+
+## Packaging
+When shipping a complete kit (rather than a single part) it should be shipped in the white 18l Really Useful Box it was provided in. Inside the box, all parts of the kit should either be placed into the jiffy bags that they were provided in or wrapped in bubble wrap. The batteries **must** be placed inside the charging bag provided with the kit. If you suspect that any batteries are damaged then do not ship them and contact <logistics@studentrobotics.org> immediately. Any remaining space in the box should be filled either with paper packaging (old Amazon parcels are a good source of this) or bubble wrap. Please do not use packing peanuts as these are messy and hard to dispose of.
```

Comment:
Do the teams know that they're meant to keep the packaging it came in? I expect most will have thrown it away unless explicitly told otherwise.